code/modules/hydroponics/trays/tray.dm

```diff
@@ -282,7 +282,7 @@
 	return TRUE
 
 //Clears out a dead plant.
-/obj/machinery/portable_atmospherics/hydroponics/proc/remove_dead(var/mob/user, var/silent)
```

Comment:
using var/ in proc parameters is normal and something we do everywhere. No purpose in creating isolated exceptions to established policy